common/usb_cdc.c

```diff
@@ -83,7 +90,12 @@ static const char cfgDescriptor[] = {
 	0x01, // bNumEndpoints
 	0x02, // bInterfaceClass:       Communication Interface Class
 	0x02, // bInterfaceSubclass:    Abstract Control Model
-	0x01, // bInterfaceProtocol:    Common AT Commands, V.25ter
+	0x00, // bInterfaceProtocol:    No protocol
+	      // While setting an interface protocol of 0xFF would be more correct
+	      // (as it is vendor-specific), Linux's cdc-acm module only matches
+	      // devices that have a well-known (modem) Interface Protocol, or 0. We
+	      // also are not a modem, so give a hint that we shouldn't get AT
+	      // commands.

```

Comment:
bmCapabilities setting of the Communication Class prevents AT commands being sent over the Communication Class interface. Our client software controls the Data Class interface and doesn't send AT commands there as well.

Before changing this setting we should be sure about the benefits. Are there any?

Reply:
ack, reverting that